core/renderers/common/marker_svg.ts

```diff
@@ -494,20 +482,20 @@ export class MarkerSvg {
    */
   protected positionPrevious_(
       width: number, markerOffset: number, markerHeight: number,
-      connectionShape: Notch|PuzzleTab) {
-    // AnyDuringMigration because:  Property 'pathLeft' does not exist on type
-    // 'Notch | PuzzleTab'.
+      connectionShape: Notch) {
```

Comment:
This is a breaking change (unless we regard the whole `MarkerSvg` class as being project-internal, maybe.)

Reply:
I don't think this is a breaking change because passing a puzzle tab wouldn't have worked anyway (b/c it doesn't have a `.pathLeft` property. I think the types were just defined incorrectly before.

Reply:
Fair enough!